tools/misbehaving-jmx-server/src/main/java/org/datadog/misbehavingjmxserver/App.java

```diff
@@ -42,6 +48,11 @@ class AppConfig {
     // Can only be set via env var
     public int controlPort = Defaults.JMXSERVER_CONTROL_PORT;
 
+    @Parameter(names = {"--config-path", "-cfp"})
+    public String config_path = "configs/benchmark_config.yaml";
```

Comment:
```suggestion
    public String config_path = "./misbehaving_jmx_config.yaml";
```

Lets default it to the current directory